src/webextension/list/containers/item-filter.css

```diff
@@ -0,0 +1,51 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+.filter {
+  border: 1px solid #b6b6b6;
+  position: relative;
+  margin: 12px;
+}
+
+.filter:focus-within,
+.filter:focus-within:hover {
+  box-shadow: none;
```

Comment:
This should probably be removed. It used to exist in `app.css` because things looked very broken otherwise. I don't think it should be an issue now.